crawl-ref/source/mon-data.h

```diff
@@ -7430,6 +7457,21 @@ DUMMY(MONS_HELL_LORD, '&', COLOUR_UNDEF, "hell lord", TILEP_MONS_PROGRAM_BUG)
     {TILEP_MONS_AMAEMON}, TILE_ERROR
 },
 
+{
+    MONS_GERALT, '6', LIGHTGREY, "Geralt",
+    M_UNIQUE | M_SEE_INVIS | M_FAST_REGEN | M_WARM_BLOOD 
+        | M_MALE | M_FAST_REGEN | M_TWO_WEAPONS,
+    MR_RES_POISON,
+    25, MONS_DEMONSPAWN, MONS_DEMONSPAWN, MH_NATURAL, 80,
+    { {AT_HIT, AF_PLAIN, 25}, {AT_HIT, AF_PLAIN, 25}, AT_NO_ATK, AT_NO_ATK },
```

Comment:
Damage here feels a little low. This is in the same ballpark as a vampire knight or deep troll, which are common Depths enemies - it seems like we'd want a unique to be at least a little more threatening.